src/natcap/invest/seasonal_water_yield/seasonal_water_yield_core.pyx

```diff
@@ -101,76 +105,77 @@ cpdef calculate_local_recharge(
     flow_dir_raster_info = pygeoprocessing.get_raster_info(flow_dir_mfd_path)
     flow_dir_nodata = flow_dir_raster_info['nodata'][0]
     raster_x_size, raster_y_size = flow_dir_raster_info['raster_size']
+
     cdef ManagedFlowDirRaster flow_raster = ManagedFlowDirRaster(
-        flow_dir_mfd_path, 1, 0)
+        flow_dir_mfd_path.encode('utf-8'), 1, 0)
+    cdef NeighborTuple neighbor
 
     # make sure that user input nodata values are defined
     # set to -1 if not defined
     # precipitation and evapotranspiration data should
     # always be non-negative
-    et0_m_raster_list = []
+    cdef vector[ManagedRaster] et0_m_rasters
     et0_m_nodata_list = []
-    for et0_path in et0_path_list:
-        et0_m_raster_list.append(_ManagedRaster(et0_path, 1, 0))
+    for i, et0_path in enumerate(et0_path_list):
+        et0_m_rasters.push_back(ManagedRaster(et0_path.encode('utf-8'), 1, 0))
```

Comment:
Is the enumeration still needed?  It looks like there are a few more enumerations in the next 30 lines or so

Reply:
Not needed!